.github/workflows/python_tools.yml

```diff
@@ -31,6 +31,14 @@ jobs:
         python -m pip install pylint
         
      #---------- Ontology Explorer ----------#
+    - name: Ontology Validator Install dependencies
```

Comment:
Not sure I follow this change here. It is already in the file, why re-adding it?

Reply:
Only the ontology validator was being setup, but there are some methods from the instance validator that I need to access as well so I added that set up.

Reply:
any updates on this?